config/sets/doctrine25.php

```diff
@@ -3,13 +3,14 @@
 declare(strict_types=1);
 
 use PHPStan\Type\ObjectType;
+
```

Comment:
we may need a `CodingStyle` rector rule for it when auto import enable, on same `use FullyQualifiedName` next `use FulyQualifiedName`